db/migrate/20190306114813_create_account_recovery_requests.rb

```diff
@@ -0,0 +1,18 @@
+class CreateAccountRecoveryRequests < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def up
+    create_table :account_recovery_requests do |t|
+      t.integer :user_id, null: false
+      t.string :request_token, null: false
+      t.datetime :requested_at, null: false
+      t.timestamps
+    end
+    add_index :account_recovery_requests, %i[user_id], unique: true, using: :btree
+    add_index :account_recovery_requests, %i[request_token], unique: true, using: :btree
+  end
+
+  def down
+
```

Comment:
We're gonna want a migration to drop this table here

Reply:
Good catch that was a dev version I was using to redo the migration